crates/core/tedge_agent/src/agent.rs

```diff
@@ -20,6 +21,7 @@ use tracing::info;
 use tracing::instrument;
 
 const TEDGE_AGENT: &str = "tedge-agent";
+const TEDGE_TO_TE_CONVERTER: &str = "tedge-to-te-converter";
```

Comment:
I see no point for a `const`. This name is used in a single place and is internal to the tedge-to-te-converter.

Reply:
removed